OrbitGl/DataManager.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_DATA_MANAGER_H_
+#define ORBIT_GL_DATA_MANAGER_H_
+
+#include <thread>
+
+#include "ProcessData.h"
+#include "absl/container/flat_hash_map.h"
+
+// This class is responsible for storing and
+// navigating data on the client side. Note that
+// every method of this class should be called
+// on the main thread.
+class DataManager final {
```

Comment:
So this class would contain all data displayed by the ui?

Reply:
Yes, this is the idea.